tools/release/image-binary-release/delete-docker-image-tag.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
```

Comment:
Yes @sethAmazon  this can be written in go but it would be more easier this way and also its hard for me to find any reason to use the package main go to write a similar script.